### PR TITLE
bib-tool 2.61

### PR DESCRIPTION
**Pull request by adamliter**
The problem in the BibTool tarball from version 2.60 is gone in version 2.61, so the workaround discussed in https://github.com/Homebrew/homebrew/issues/40559 is no longer needed.

So I've deleted the special download strategy for the new version of BibTool.

Also, the pull request (https://github.com/Homebrew/homebrew/pull/40574) that I submitted to fix https://github.com/Homebrew/homebrew/issues/40559 introduced a problem when trying to install BibTool on Linux with Linuxbrew, since Linux doesn't use `bsdtar`, and the `--fast-read` option that was introduced as a workaround is only available in `bsdtar`, not `gnutar`. That was shortsighted on my part. Sorry about that.

This was mentioned in https://github.com/Homebrew/linuxbrew/issues/456 and ultimately fixed with a pull request to Linuxbrew (https://github.com/Homebrew/linuxbrew/pull/457).

In other words, the formulas for BibTool v. 2.60 diverged between Homebrew and Linuxbrew. This new formula for v. 2.61 works with both Homebrew and Linuxbrew (tested on Mac OS 10.10.4 and Ubuntu v. 14.04). I'm not sure how formulas are maintained across Homebrew and Linuxbrew, but there is no longer any need for them to diverge, so this new formula can be used in Linuxbrew, too. Should I submit a separate pull request to Homebrew/Linuxbrew? I don't think the commit can be straight-forwardly cherry-picked, since the formula in Linuxbrew is currently different.

Thanks! :smile:

### CODE REVIEW

**Comment by dunn** [comment]
About once a week @sjackman merges the changes from Homebrew into Linuxbrew; I don't know if it would be easier for him to reconcile the formulae with a PR.

**Comment by sjackman** [comment]
No pull request is necessary. I'll pick up the change on the next weekly merge.

**Comment by adamliter** [comment]
Sounds good; thanks @sjackman!

**Comment by DomT4** [comment]
:tada: for getting rid of the download strategy.

Merged in https://github.com/Homebrew/homebrew/commit/8253d308de5a6f1d1d91017f85a31de4977bb506. Thank you for your contribution to Homebrew, we appreciate it! :smiley_cat:
